StudyHub/StudyHub/StudyHub.csproj

```diff
@@ -0,0 +1,18 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+	<PropertyGroup>
+		<TargetFramework>net7.0</TargetFramework>
+		<Nullable>enable</Nullable>
```

Comment:
```suggestion
		<Nullable>disabled</Nullable>
```

Reply:
no, it has to be enable to use nullable types whenever in project